src/practitest_firecracker/practitest.clj

```diff
@@ -10,7 +10,7 @@
 
 ;; ===========================================================================
 ;; api version
-(def ^:const fc-version "1.4.0")
+(def ^:const fc-version "0.5.1")
```

Comment:
Did we already release 1.4.0? If so, we cannot go back and should always increase the version number.

Reply:
If the version corresponds to release than no.